src/NServiceBus.Transport.RabbitMQ/Connection/ConfirmsAwareChannel.cs

```diff
@@ -200,13 +207,15 @@ void SetException(ulong key, string exceptionMessage)
             }
         }
 
-
         public void Dispose()
         {
+            ClearMessages();
```

Comment:
This change is unnecessary. Calling `Dispose` on an open channel will call `Channel_ModelShutdown` on it. I'm going to revert this.